apps/api/src/app/workflows-v2/workflow.controller.ts

```diff
@@ -114,4 +117,17 @@ export class WorkflowController {
       })
     );
   }
+
+  @Post('/:workflowId/step/:stepUuid/preview')
+  @UseGuards(UserAuthGuard)
+  async generatePreview(
+    @UserSession() user: UserSessionData,
+    @Param('workflowId') workflowId: string,
+    @Param('stepUuid') stepUuid: string,
```

Comment:
We should use the `stepId` here. What's the stepUuid?

Reply:
we shouldn't we are trying to preview a specific unique step, it's a part of the ids conversations in slack, that's a unuiqe id.